core/src/main/java/io/opencensus/tags/Tags.java

```diff
@@ -49,6 +49,35 @@ public static TagPropagationComponent getTagPropagationComponent() {
     return tagsComponent.getTagPropagationComponent();
   }
 
+  /**
+   * Returns {@code true} if a tagging implementation is available.
+   *
+   * <p>When tagging is not available, calling {@link #setState} has no effect.
+   *
+   * @return {@code true} if a tagging implementation is available.
+   */
+  public static boolean isTaggingAvailable() {
+    return tagsComponent.isTaggingAvailable();
+  }
+
+  /**
+   * Returns the current {@code TaggingState}.
+   *
+   * @return the current {@code TaggingState}.
+   */
+  public static TaggingState getState() {
```

Comment:
Document that always returns DISABLE if {@link #isImplementationAvailable()} returns false.

Reply:
Done